utils/mux_gen.py

```diff
@@ -207,11 +204,6 @@ def normpath(p, to=None):
     # XML Files can't have "--" in them, so instead we use ~~
     xml_comment = generated_with.replace("--", "~~")
 
-    # ------------------------------------------------------------------------
-    # Create a makefile to regenerate files.
```

Comment:
@mithro Do we still want a way to output the parameters that were used to a file or stderr?

Reply:
Not sure what you are suggesting here.  I could output the parameters used to define the mux into the verilog file.  Would that match your request?

Reply:
@elmsfu Was suggesting that the tool have a way to generate the `CMakeLists.txt` file as the output.